### DISCUSSION TITLE
NordPass

### DISCUSSION THREAD

**Discussion by giscus** [AwaitQuality/windowsonarm#46] [General]
# NordPass

Check if NordPass is ARM ready on Windows and can be used on Windows ARM devices such as the Surface with the Snapdragon X Elite.

https://windowsonarm.org/d14bc720-31f7-4452-98b9-fd537b210e22

**Comment by adelvo-dev** [comment]
Chrome Extension works on Chrome:
Version 129.0.6668.90 (Offizieller Build) (arm64)
MacOS - UTM VM - Windows11 for Arm 24H2
